src/install/java/download/aws.mdx

```diff
@@ -0,0 +1,10 @@
+---
+componentType: default
+headingText: Choose your Lambda installation
+---
+
+We offer in-depth performance monitoring for your serverless [AWS Lambda](https://aws.amazon.com/lambda/) functions. 
+
+This Lambda monitoring feature is not the same as the [Lambda monitoring integration](/docs/integrations/amazon-integrations/aws-integrations-list/aws-lambda-monitoring-integration)**, which only uses CloudWatch data. Serverless Lambda monitoring is more extensive and uses both CloudWatch data and code-level instrumentation to deliver more in-depth monitoring.
```

Comment:
Not sure if this is because I lacked the context, but for a minute I didn't realize you were comparing two different things. I'm not sure if it's because you linked the integration, but my mind clicked into "Oh okay, guess i have to read about the integrations because that's what i'm installing." But you're saying what IS being installed is the Lambda monitoring feature, not the integration.

In any case, differentiating these two could be helpful.

Reply:
I agree!